vendor.conf

```diff
@@ -23,7 +23,7 @@ github.com/RackSec/srslog 456df3a81436d29ba874f3590eeeee25d666f8a5
 github.com/imdario/mergo 0.2.1
 
 #get libnetwork packages
-github.com/docker/libnetwork 1a019214c9cb80bd56219e5d6994a22caf302895
+github.com/docker/libnetwork bb4fe0b6f806412fc671348bfdc42a693c00a0ae
```

Comment:
@mavenugo Updated the vendoring to libnetwork SHA 4610dd67c7b9828bb4719d8aa2ac53a7f1f739d2 (brings https://github.com/docker/libnetwork/pull/1354)